### PR TITLE
Correct filtering of percent bases

### PR DESCRIPTION
**Pull request by jsangmeister**

### CODE REVIEW

**Comment by tsiegleauq** [comment]
fixed tests

File: client/src/app/site/polls/components/poll-form/poll-form.component.spec.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { E2EImportsModule } from 'e2e-imports.module';
 
 import { PollFormComponent } from './poll-form.component';
 
-describe('PollFormComponent', () => {
+fdescribe('PollFormComponent', () => {
```
Comment (jsangmeister) [inline client/src/app/site/polls/components/poll-form/poll-form.component.spec.ts]: Is the fdescribe on purpose? Should probably be reverted if not to not hassle woth debugging other tests.

Comment (tsiegleauq) [reply client/src/app/site/polls/components/poll-form/poll-form.component.spec.ts]: no its not :D thanks

Comment (tsiegleauq) [reply client/src/app/site/polls/components/poll-form/poll-form.component.spec.ts]: #5238 is already on top of this PR and has the tests functional again.

**Pull request merged**